swagger-spec/registrations/definition.yaml

```diff
@@ -141,6 +144,19 @@ properties:
         type: string
         readOnly: true
         description: 'The reasoning for why this registration was withdrawn, as entered by the administrator that initiated the withdrawal.'
+      draft_registration:
+        type: string
+        readOnly: false
+        description: 'The unique ID of the draft registration from which the registration was created.'
+      registration_choice:
+        type: string
+        readOnly: false
+        description: 'Type of registration you want to choose to be, immediate or embargo. If you choose to be embargo, you need to provide with a lift_embargo datetime.'
+      lift_embargo:
+        type: string
+        format: date-time
+        readOnly: false
+        description: 'The time at which the registered node will become public only avaible when you choose embargo for registration_choice.'
```

Comment:
Rewording suggestion: A future datetime when the registration will become public.  This field should be set when "registration_choice" is set to "embargo".